Platform/Shared/VMRemovableDrivesView.swift

```diff
@@ -31,44 +31,43 @@ struct VMRemovableDrivesView: View {
     }
     
     init(vm: UTMVirtualMachine) {
-        self.vm = vm
+        self._vm = State<UTMVirtualMachine>(initialValue: vm)
         self.config = vm.configuration
         self.sessionConfig = vm.viewState
     }
     
     var body: some View {
-        Group {
+        return Group {
```

Comment:
Is there a reason for the added `return` here?

Reply:
No, there isn't. I'll remove it